src/main/resources/application.properties

```diff
@@ -1 +1 @@
-
+getstore.url = https://fakestoreapi.com
```

Comment:
application.properties i application.yaml to te same pliki tylko w innym formacie
Wybierz jeden (yaml jest nowszy i bardziej używany) i do niego dodaj wszystkie parametry.